playbooks/roles/deploy-osh/templates/keystone.yaml.j2

```diff
@@ -1,2 +1,40 @@
 ---
-{{ lookup('template','files/common-images-overrides.yml') | from_yaml | to_nice_yaml(indent=2) }}
+{% if developer_mode %}
+conf:
+  apache2:
+    binary: apache2ctl
+    start_parameters: -DFOREGROUND -k start
+    site_dir: /etc/apache2/vhosts.d
+    conf_dir: /etc/apache2/conf.d
+    security: |
+      <Directory "/var/www">
+      Options Indexes FollowSymLinks
+      AllowOverride All
+      <IfModule !mod_access_compat.c>
+        Require all granted
+      </IfModule>
+      <IfModule mod_access_compat.c>
+        Order allow,deny
+        Allow from all
+      </IfModule>
+      </Directory>
```

Comment:
We also need
```
    a2enmod:
      - mod_version
```

since there's a `IfVersion` check in wsgi-keystone.conf and the module is apparently not included by default

Reply:
Good catch. I think we may need to include that in all of our apache related overrides as it seems to be used all around.

Reply:
FYI `version` and `mod_version` both seems to work

Reply:
Yes, I think it will be needed in all similar PR's